### PR TITLE
Add view all links

### PR DESCRIPTION
**Pull request by iomodo**
#### Summary
This PR adds view all links to the LHS

https://user-images.githubusercontent.com/2340127/178324588-280c2d36-6d98-452e-95ba-50ddc756a846.mp4

#### Ticket Link

  Fixes: https://mattermost.atlassian.net/browse/MM-44751

#### Checklist

- [ ] Telemetry updated
- [ ] Gated by experimental feature flag
- [ ] Unit tests updated

### CODE REVIEW

File: webapp/src/components/sidebar/sidebar.tsx [hunk 1]
```diff
@@ -28,6 +28,7 @@ export interface SidebarGroup {
     display_name: string;
     collapsed: boolean;
     items: Array<GroupItem>;
+    customLastItem?: ReactNode;
```
Comment (crspeller) [inline webapp/src/components/sidebar/sidebar.tsx:31]: Maybe `afterList` or `afterGroup`? It's not really adding a last item to the group it's just flexible space afterwards.

**Review by crspeller — APPROVED** [review]
Just one minor suggestion.

File: webapp/src/components/sidebar/playbooks_sidebar.tsx [hunk 2]
```diff
@@ -47,9 +49,57 @@ const PlaybooksSidebar = (props: PlaybookSidebarProps) => {
...
+            } else if (groups[i].id === 'playbooksCategory') {
+                groups[i].customLastItem = viewAllPlaybooks();
+            }
+        }
+    };
+
+    const {formatMessage} = useIntl();
+    const viewAllMessage = formatMessage({defaultMessage: 'View all...'});
+
+    const viewAllRuns = (): ReactNode => {
```
Comment (agarciamontoro) [inline webapp/src/components/sidebar/playbooks_sidebar.tsx:69]: Nit: is this explicit typing needed?
```suggestion
    const viewAllRuns = () => {
```

File: webapp/src/components/sidebar/playbooks_sidebar.tsx [hunk 3]
```diff
@@ -47,9 +49,57 @@ const PlaybooksSidebar = (props: PlaybookSidebarProps) => {
...
+                >
+                    <ItemDisplayLabel>
+                        {viewAllMessage}
+                    </ItemDisplayLabel>
+                </StyledLink>
+            </ItemContainer>
+        );
+    };
+
+    const viewAllPlaybooks = (): ReactNode => {
```
Comment (agarciamontoro) [inline webapp/src/components/sidebar/playbooks_sidebar.tsx:86]: Same nit here
```suggestion
    const viewAllPlaybooks = () => {
```

**Review by agarciamontoro — APPROVED** [review]
Code looks perfect!

**Review by abhijit-singh — APPROVED** [review]
Looks great @iomodo!

One minor, non-blocking point for the on-click state that I'm guessing needs to be fixed for all LHS elements yet — for the `View all` option, I don't think we need an `active` state. If you click on it and hover away, it should go back to the default state instead of appearing `active`

**Pull request merged**